src/storage/v2/disk/storage.cpp

```diff
@@ -1368,7 +1368,7 @@ DiskStorage::DiskAccessor::CheckVertexConstraintsBeforeCommit(
 /// std::map<dst_vertex_gid, ...>
 /// Here we also do flushing of too many things, we don't need to serialize edges in read-only txn, check that...
 [[nodiscard]] utils::BasicResult<StorageManipulationError, void> DiskStorage::DiskAccessor::FlushModifiedEdges(
-    const auto &edge_acc) {
+    auto &edges_acc) {
```

Comment:
Why did we lose `const`? Do we need to use `.cend()`?

Reply:
@andrejtonev, @as51340 and I played with this for a while, but couldn't figure out a nice solution, could you try that out and propose something that will work (I think we tried to force `cend` and it also didn't work...

Reply:
const added back here